examples/tsconfig.json

```diff
@@ -1,12 +1,4 @@
 {
   "extends": "../tsconfig.json",
```

Comment:
@himself65 do the changes in `examples` also work with just using `npm` (no monorepo)

Reply:
Yes of course, I remove this only because I have `extends` on the top